core-services/prow/02_config/_labels.yaml

```diff
@@ -604,6 +604,11 @@ repos:
         name: docs-approved
         target: both
         addedBy: label
+      - color: 00D8FF
+        description: Related to insights plugin
+        name: component/insights
```

Comment:
The other component labels are alphabetized. Can we move this after kubevirt above?

Reply:
Done